ui/app/AppLayouts/Chat/components/NicknamePopup.qml

```diff
@@ -80,11 +85,16 @@ ModalPopup {
         //% "Done"
         label: qsTrId("done")
         anchors.bottom: parent.bottom
-        disabled: popup.nicknameLength === 0 || popup.nicknameTooLong
+        disabled: popup.nicknameTooLong
         onClicked: {
-            if (!userName.startsWith("@")) {
-                // Replace username only if it was not an ENS name
-                userName = nicknameInput.textField.text
+            if (!isEnsVerified) {
+                // Change username title only if it was not an ENS name
+                 if (nicknameInput.textField.text === "") {
+                     // If we removed the nickname, go back to showing the alias
+                     userName = alias
```

Comment:
Where does the property `userName` come from? IOW, when `userName` is changed here, what is the effect?

Reply:
`username` is the name of the property that is used by `NicknamePopup` for its title. I'll add it as a property because there is a chance that NicknamePopup could be used by something else